sentry_sdk/api.py

```diff
@@ -207,7 +207,8 @@ def start_span(
 @hubmethod
 def start_transaction(
     transaction=None,  # type: Optional[Transaction]
+    custom_sampling_context=None,  # type: Optional[Dict[str, Any]]
```

Comment:
Is there a need for this change? This would just end up in kwargs

Reply:
So it shows up in the automatically-generated docs? But can demote it if you'd like.

Reply:
feel like we need a different solution, btw docstring already links to the wrapped method

Reply:
Good to know. Anyway, I've already pulled it into kwargs.